lib/usd/translators/mayaReferenceUpdater.cpp

```diff
@@ -191,67 +203,53 @@ UsdMayaPrimUpdater::PushCopySpecs PxrUsdTranslators_MayaReferenceUpdater::pushCo
         return PushCopySpecs::Failed;
     }
 
-    // Use the edit router to find the destination layer and path.
-    auto dstEditRouter = MayaUsd::getEditRouter(TfToken("mayaReferencePush"));
-    if (!dstEditRouter) {
-        return PushCopySpecs::Failed;
-    }
-
     PXR_NS::VtDictionary routerContext = getContext()->GetUserArgs();
+
+    // Pass the source and destination stage, layer and path to routers.
```

Comment:
These values are required for the copy-transform code. In general, an edit router might want to know these detail to make decision where to route.